spec/controllers/course/achievement_condition_levels_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+RSpec.describe Course::Achievement::Condition::LevelsController, type: :controller do
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let!(:user) { create(:administrator) }
```

Comment:
try not to use let! since let is lazy. if you need it created, then obviously do it. but user is referenced in sign_in, so it should not be necessary.